.php-cs-fixer.dist.php

```diff
@@ -72,6 +72,7 @@
     'ternary_operator_spaces'             => true,
     'trailing_comma_in_multiline'         => true,
     'whitespace_after_comma_in_array'     => true,
+    'import_functions'                    => true,
```

Comment:
See: https://github.com/pimcore/studio-backend-bundle/issues/200#issuecomment-2197184449

Reply:
Thanks for the comment! I will add it!

Reply:
You can still use `'global_namespace_import' => ['import_functions' => true],` if you prefer imports to backslashes before every function call.

Reply:
yeah. I just readded it :) Thanks for the hint though. With all the new bundles we try to rather do use statements.

Reply:
There's also [`ordered_imports`](https://mlocati.github.io/php-cs-fixer-configurator/#version:3.59|fixer:ordered_imports) if you want to separate/group class/const/function imports.

Reply:
even better, yeah i find it not so nice if they are mixed. thank you very much!